src/Homie/Config.cpp

```diff
@@ -49,6 +47,17 @@ bool Config::load() {
   JsonObject& parsedJson = jsonBuffer.parseObject(buf);
   if (!parsedJson.success()) {
     Interface::get().getLogger() << F("✖ Invalid JSON in the config file") << endl;
+    return JsonObject::invalid();
+  }
+  return parsedJson;
+}
+
+
+bool Config::load() {
+  _valid = false;
+  JsonObject& parsedJson = Config::loadJSONObject();
+  if (!parsedJson.success()) {
```

Comment:
It's been a decade or two since I've done any real C++ coding.  Thanks for the sanity check.